app/templates/gentelella/guest/event/base.html

```diff
@@ -156,13 +156,14 @@ <h4 style="font-weight: 300;"><i class="glyphicon glyphicon-map-marker"></i> {{
         </div>
     </div>
     <div class="row">
-        <div class="col-md-8 col-sm-8 col-xs-12 col-md-push-1 col-sm-push-2 event-info-container-holder"
+        <div class="col-xs-12 col-md-push-1 col-sm-push-2 event-info-container-holder {% if active_page =='sessions' or active_page == 'schedule' %}col-md-10 col-sm-10 {% else %}col-md-8 col-sm-8{% endif %}" 
```

Comment:
And keep the length of a line very long. Try to wrap the single line into multiple lines. makes it more readable